### PR TITLE
47308 MSI Home page

### PR DESCRIPTION
**Pull request by mubeenMastek**

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,100 +1,28 @@
...
+        <div class="col-lg-9">
+            <h1 id="headingLevelOne" class="mt-0">Maritime Safety Information</h1>
+            <p class="intro-text">View the latest safety-critical navigational information </p>
+            <p class="intro-text">If you are an ADMIRALTY Distributor, sign in to see more content </p>
+            <div class="row mb-5">
+                <div class="col-md-6 pe-md-0">
+                    <div class="nm-link">
+                        <h2>Notice to Mariners (NMs)</h2>
+                        <p>  Weekly updates that provide bridge crews with the latest safety-critical navigational information. They can be downloaded for free here, or purchased as weekly bulletins from your ADMIRALTY Distributor.</p>
+                        <a href="@Url.Action("Index", "NoticesToMariners")" class="bottom-link">Go to Notice to Mariners</a>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml:15]:
```suggestion
                        <a href="@Url.Action("Index", "NoticesToMariners")" class="bottom-link">Go to Notices to Mariners</a>
```

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,100 +1,28 @@
...
-                    <p class="text">
-                        NAVAREA I and UK Coastal navigational warnings reproduced online.
-                    </p>
-
-                    <div class="cta">
-                        <h4>Go to Radio Navigational Warnings</h4>
+                <div class="col-md-6 ps-md-0">
+                    <div class="rnw-link">
+                        <h2>Radio Navigational  Warnings (RNWs)</h2>
+                        <p>NAVAREA 1 and UK Coastal navigation warnings reproduced online</p>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml:21]: Add dot/punctuation mark at end to make it consistent with NM tab.
```suggestion
                        <p>NAVAREA 1 and UK Coastal navigation warnings reproduced online.</p>
```

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -1,100 +1,28 @@
...
-                    </p>
-
-                    <div class="cta">
-                        <h4>Go to Notices to Mariners</h4>
+<div class="container-fluid">    
+    <div class="row justify-content-lg-center">
+        <div class="col-lg-9">
+            <h1 id="headingLevelOne" class="mt-0">Maritime Safety Information</h1>
+            <p class="intro-text">View the latest safety-critical navigational information </p>
+            <p class="intro-text">If you are an ADMIRALTY Distributor, sign in to see more content </p>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml:9]:
```suggestion
            <p class="intro-text">If you are an ADMIRALTY Distributor, please sign in to see more content </p>
```

**Review by usha13529 — CHANGES_REQUESTED** [review]
Few minor review comments to address please.

File: Tests/AutoTests/FunctionalTest/functional-homepage.spec.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  let home: homePage;
+  test.beforeEach(async ({ page }) => {
+    await page.goto(app.url);
+    home=new homePage(page);
+  });
+
+  test('Should go to Home Page', async ({ page, context }) => {
+    
+    await home.verifyAdmiraltyHomePage();
+    await home.verifyadmiraly();
```
[the remark this answers is not in the document]
Comment (Pradeep15558) [reply Tests/AutoTests/FunctionalTest/functional-homepage.spec.ts]: Resolved

File: Tests/pageObject/Home.page.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  public async verifyAdmiraltyHomePage() 
+  {
+  expect(await this.siteTitle.innerText()).toContain('Maritime Safety Information');
+  }
+
+  public async verifyHomePageTitle() 
+  {
+  expect(await this.headingLevelOne.innerText()).toContain('Maritime Safety Information')
+  } 
+  public async verifyadmiraly()
```
[the remark this answers is not in the document]
Comment (Pradeep15558) [reply Tests/pageObject/Home.page.ts]: Resolved

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml [hunk 6]
```diff
@@ -1,100 +1,28 @@
...
+            <div class="row mb-5">
+                <div class="col-md-6 pe-md-0">
+                    <div class="nm-link">
+                        <h2>Notice to Mariners (NMs)</h2>
```
[the remark this answers is not in the document]
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml]: Changes done.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml]: Changes done.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml (hunk 2 above)
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml]: Changes done.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml (hunk 3 above)
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml]: Changes done.

**Review by usha13529 — APPROVED** [review]
Thanks @mubeenMastek & @Pradeep15558 for changes. Looks good to me.

**Review by nevillejrbrown — APPROVED** [review]

**Review by Chownder — APPROVED** [review]
LGTM, thanks @mubeenMastek 

p.s. I can see that the beta banner has been removed when running locally, but darned if I can see it in the changes.

**Pull request merged**